### PR TITLE
arm: 96boards: Add Chardev support for relevant boards

### PR DESCRIPTION
**Pull request by Mani-Sadhasivam**
As of now both Dragonboard410c and Hikey boards run latest LTS kernel.
Hence, adding chardev support for those.

Along with this support, clang-format has been executed for the source file.

Signed-off-by: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>

### CODE REVIEW

**Comment by Mani-Sadhasivam** [comment]
@arfoll Can you please review and merge this PR? I still have many PR's in queue for 96Boards.

File: src/arm/96boards.c [hunk 1]
```diff
@@ -47,12 +47,21 @@ int db410c_ls_gpio_pins[MRAA_96BOARDS_LS_GPIO_COUNT] = {
     36, 12, 13, 69, 115, 4, 24, 25, 35, 34, 28, 33,
 };
 
+int db410c_chardev_map[MRAA_96BOARDS_LS_GPIO_COUNT][2] = {
+    { 0, 36 }, { 0, 12 }, { 0, 13 }, { 0, 69 }, { 0, 115 }, { 0, 129 },
```
Comment (Propanu) [inline src/arm/96boards.c]: Does line 129 map to gpio4? Everything else besides this is identical.

Comment (Mani-Sadhasivam) [reply src/arm/96boards.c]: That specific pin comes from PMIC, hence I assumed that it belongs to the same gpiochip. But further investigation reveals that it belongs to Pin 3 of gpiochip 2. Will change it.

**Comment by alext-mkrs** [comment]
I wouldn't recommend mixing the styling changes with functional ones (especially at this scale I see) - it makes it really hard to analyze the file history.

The chardev change itself seems to be ok by inspection.

**Comment by alext-mkrs** [comment]
Thanks! LGTM now, :+1: from me for merging.

**Comment by malikabhi05** [comment]
Merged, Thanks!

**Pull request closed without merging**